just remote results in `No matching version found for @mutinywallet/ui@*`

❯ just remote
pnpm unlink "@mutinywallet/mutiny-wasm" && pnpm install
 ERR_PNPM_NO_MATCHING_VERSION_INSIDE_WORKSPACE  In : No matching version found for @mutinywallet/ui@* inside the workspace

This error happened while installing a direct dependency of /data/Dev/Mutiny/mutiny-web
error: Recipe `remote` failed on line 10 with exit code 1

--- a/justfile
+++ b/justfile
@@ -7,7 +7,7 @@ local:
     pnpm link --global "@mutinywallet/mutiny-wasm"
 
 remote:
-    pnpm unlink "@mutinywallet/mutiny-wasm" && pnpm install
+    pnpm unlink --filter "@mutinywallet/mutiny-wasm" && pnpm install
 
 test:
     pnpm exec playwright test

